### PR TITLE
Make failed login message configurable.

### PR DESCRIPTION
**Pull request by briand44**

### CODE REVIEW

**Comment by brandonneil** [comment]
I can confirm that the functionality works as expected. I have not yet had time to do a code review.

File: uiowa_acquia_authentication.module [hunk 1]
```diff
@@ -21,8 +41,13 @@ function uiowa_acquia_authentication_simplesamlphp_auth_allow_login($attributes)
...
     return TRUE;
   }
+
+  $message = variable_get('uiowa_acquia_authentication_failed_login_message', array(
+    'value' => 'We are sorry, your user account cannot be created.',
+    'format' => 'filtered_html'));
+
   // Fail register and login.
-  drupal_set_message(t('We are sorry, your user account cannot be created.'), 'error');
+  drupal_set_message($message['value'], 'error');
```
Comment (brandonneil) [inline uiowa_acquia_authentication.module]: I think we'll want to run this through [check_markup()](https://api.drupal.org/api/drupal/modules%21filter%21filter.module/function/check_markup/7.x)

File: uiowa_acquia_authentication.module [hunk 2]
```diff
@@ -21,8 +41,13 @@ function uiowa_acquia_authentication_simplesamlphp_auth_allow_login($attributes)
   if (!empty($roles)) {
     return TRUE;
   }
+
+  $message = variable_get('uiowa_acquia_authentication_failed_login_message', array(
+    'value' => 'We are sorry, your user account cannot be created.',
+    'format' => 'filtered_html'));
```
Comment (brandonneil) [inline uiowa_acquia_authentication.module]: Ending closing parenthesis and semicolon should be on new line.

File: uiowa_acquia_authentication.module [hunk 3]
```diff
@@ -7,6 +7,26 @@
...
 
 /**
+ * Implements hook_menu().
+ */
+function uiowa_acquia_authentication_menu() {
+  $items = array();
+
+  $items['admin/config/people/simplesamlphp_auth/uiowa_acquia_authentication'] = array(
+    'title' => 'UIowa Acquia authentication settings',
+    'description' => 'Description.',
```
Comment (richardbporter) [inline uiowa_acquia_authentication.module]: A better description would be nice. :)

**Comment by brandonneil** [comment]
@richardbporter the permission is defined by the simplesamlphp_auth module and we're piggybacking off the permission instead of defining our own.

**Comment by briand44** [comment]
I don't think we need to use check_markup() anywhere since we are using a text format that should handle stripping out html that is not allowed. I do believe we need to translate it though.

**Pull request merged**